### PR TITLE
Hunyang/dispatcher fix cv index bug

### PR DESCRIPTION
**Pull request by hcyang**

### CODE REVIEW

File: common/config/rush/pnpm-lock.yaml [hunk 1]
```diff
@@ -31,6 +26,8 @@ dependencies:
   antlr4: 4.7.2
   applicationinsights: 1.6.0
   await-delay: 1.0.0
+  botbuilder-lg: 4.7.0-preview
+  botframework-expressions: 4.7.0-preview
```
Comment (daveta) [inline common/config/rush/pnpm-lock.yaml:30]: >botframework-expressions: 4.7.0-preview [](start = 2, length = 39)

Are we supposed to add 4.7.0-preview stuff in here?

Comment (hcyang) [reply common/config/rush/pnpm-lock.yaml:31]: Ignore the pnpm-lock file, it's auto generated. I did not reference those two packages and don't know how they got there, must have come from 'rush update'

**Comment by daveta** [comment]
>     public static loadLabelTextScoreFile(

Seems like there are alot of different formats.  We did something a bit layered in python where you can build on something that just parses tsvs and then layer stuff on top.  For example, seems like you have part of it - but seems like loadLabelTextScoreContent could be just an array instead of separate function.  Just a thought.

---
Refers to: packages/dispatcher/src/utility/utility.ts:790 in 12143e8. [](commit_id = 12143e83df81e161af8ddffa22532546d77c3da1, deletion_comment = False)

File: packages/dispatcher/src/model/language_understanding/featurizer/NgramSubwordFeaturizer.ts [hunk 2]
```diff
@@ -154,10 +157,11 @@ export class NgramSubwordFeaturizer {
...
 
     public createIntentUtteranceSparseIndexArrays(
-        intentsUtterances: { "intents": string[], "utterances": string[] }):
+        intentsUtterancesWeights: { "intents": string[], "utterances": string[], "weights": number[] }):
         { "intentLabelIndexArray": number[], "utteranceFeatureIndexArrays": number[][] } {
-        const intents: string[] = intentsUtterances.intents;
-        const utterances: string[] = intentsUtterances.utterances;
+        const intents: string[] = intentsUtterancesWeights.intents;
+        const utterances: string[] = intentsUtterancesWeights.utterances;
+        const weights: number[] = intentsUtterancesWeights.weights;
```
Comment (daveta) [inline packages/dispatcher/src/model/language_understanding/featurizer/NgramSubwordFeaturizer.ts:164]: >weights [](start = 14, length = 7)

I'm not sure where this is used?

File: packages/dispatcher/src/model/language_understanding/featurizer/NgramSubwordFeaturizer.ts [hunk 3]
```diff
@@ -192,12 +196,13 @@ export class NgramSubwordFeaturizer {
...
-        intentsUtterances: { "intents": string[], "utterances": string[] },
+        intentsUtterancesWeights: { "intents": string[], "utterances": string[], "weights": number[] },
         miniBatchIndexBegin: number = 0,
         miniBatchIndexEnd: number = 0):
         { "intentLabelIndexArray": number[], "utteranceFeatureIndexArrays": number[][] } {
-        const intents: string[] = intentsUtterances.intents;
-        const utterances: string[] = intentsUtterances.utterances;
+        const intents: string[] = intentsUtterancesWeights.intents;
+        const utterances: string[] = intentsUtterancesWeights.utterances;
+        const weights: number[] = intentsUtterancesWeights.weights;
```
Comment (daveta) [inline packages/dispatcher/src/model/language_understanding/featurizer/NgramSubwordFeaturizer.ts:205]: >weights [](start = 14, length = 7)

Same here, is this used?

**Comment by cleemullins** [comment]
This PR is 90 files of complex changes, including renames code refactoring. It's got merge conflicts, failing builds, and is generally not ready for merge. 

Please refactor this into a format that's reviewable and resubmit.

**Pull request closed without merging**